src/components/modals/AccelerateTxModal.tsx

```diff
@@ -96,6 +98,14 @@ export class AccelerateTxModalComponent extends PureComponent<Props, State> {
     return feeSyntax
   }
 
+  getTxSendAmount(edgeTransaction: EdgeTransaction) {
+    // Transaction amounts are negative for send transactions
+    const nativeAmount = abs(edgeTransaction.nativeAmount)
+    // Parent network fee is used for token sends
+    const feeAmount = edgeTransaction.parentNetworkFee ?? edgeTransaction.networkFee
+    return sub(nativeAmount, feeAmount)
+  }
+
```

Comment:
Optional: This doesn't seem to access `this`, so it could be a standalone function at the bottom of the file, instead of a class method.

Reply:
Done.